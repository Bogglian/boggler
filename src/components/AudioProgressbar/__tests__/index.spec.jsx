import React from 'react';
import { shallow } from 'enzyme';
import { AudioProgressbar } from '..';

describe('AudioProgressbar Component', () => {
  describe('user possibly create a AudioProgressbar component.', () => {
    it('should render a AudioProgressbar', () => {
      const component = shallow(<AudioProgressbar />);

      expect(component).toMatchSnapshot();
    });
  });
});
