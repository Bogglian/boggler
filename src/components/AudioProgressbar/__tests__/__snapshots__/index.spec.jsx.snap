// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AudioProgressbar Component user possibly create a AudioProgressbar component. should render a AudioProgressbar 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <AudioProgressbar />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "children": <ForwardRef>
        <ForwardRef>
          <div
            className="rectangle-5"
          />
          <div
            className="rectangle-4"
          />
          <div
            className="rectangle-3"
          />
          <div
            className="rectangle-2"
          />
          <div
            className="rectangle-1"
          />
          <div
            className="rectangle-2"
          />
          <div
            className="rectangle-3"
          />
          <div
            className="rectangle-4"
          />
          <div
            className="rectangle-5"
          />
        </ForwardRef>
      </ForwardRef>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": <ForwardRef>
          <div
            className="rectangle-5"
          />
          <div
            className="rectangle-4"
          />
          <div
            className="rectangle-3"
          />
          <div
            className="rectangle-2"
          />
          <div
            className="rectangle-1"
          />
          <div
            className="rectangle-2"
          />
          <div
            className="rectangle-3"
          />
          <div
            className="rectangle-4"
          />
          <div
            className="rectangle-5"
          />
        </ForwardRef>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": Array [
            <div
              className="rectangle-5"
            />,
            <div
              className="rectangle-4"
            />,
            <div
              className="rectangle-3"
            />,
            <div
              className="rectangle-2"
            />,
            <div
              className="rectangle-1"
            />,
            <div
              className="rectangle-2"
            />,
            <div
              className="rectangle-3"
            />,
            <div
              className="rectangle-4"
            />,
            <div
              className="rectangle-5"
            />,
          ],
          "className": undefined,
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "className": "rectangle-5",
            },
            "ref": null,
            "rendered": null,
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "className": "rectangle-4",
            },
            "ref": null,
            "rendered": null,
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "className": "rectangle-3",
            },
            "ref": null,
            "rendered": null,
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "className": "rectangle-2",
            },
            "ref": null,
            "rendered": null,
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "className": "rectangle-1",
            },
            "ref": null,
            "rendered": null,
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "className": "rectangle-2",
            },
            "ref": null,
            "rendered": null,
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "className": "rectangle-3",
            },
            "ref": null,
            "rendered": null,
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "className": "rectangle-4",
            },
            "ref": null,
            "rendered": null,
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "className": "rectangle-5",
            },
            "ref": null,
            "rendered": null,
            "type": "div",
          },
        ],
        "type": Object {
          "$$typeof": Symbol(react.forward_ref),
          "attrs": Array [],
          "componentStyle": ComponentStyle {
            "componentId": "sc-dnqmqq",
            "isStatic": true,
            "rules": Array [
              "
  & {
    height: 0.8em;
    width: 100%;
    display: inherit;
    margin-top: 1.7%;

    &.none {
      display: none;
    }

    .rectangle-2 {
      -webkit-animation-delay: 0.1s;
      animation-delay: 0.1s;
    }

    .rectangle-3 {
      -webkit-animation-delay: 0.2s;
      animation-delay: 0.2s;
    }

    .rectangle-4 {
      -webkit-animation-delay: 0.3s;
      animation-delay: 0.3s;
    }

    .rectangle-5 {
      -webkit-animation-delay: 0.4s;
      animation-delay: 0.4s;
    }

    .rectangle-6 {
      -webkit-animation-delay: 0.5s;
      animation-delay: 0.5s;
    }

    .rectangle-7 {
      -webkit-animation-delay: 0.6s;
      animation-delay: 0.6s;
    }

    .rectangle-8 {
      -webkit-animation-delay: 0.7s;
      animation-delay: 0.7s;
    }

    .rectangle-9 {
      -webkit-animation-delay: 0.8s;
      animation-delay: 0.8s;
    }

    .rectangle-10 {
      -webkit-animation-delay: 0.9s;
      animation-delay: 0.9s;
    }

    .rectangle-11 {
      -webkit-animation-delay: 1s;
      animation-delay: 1s;
    }

    .rectangle-12 {
      -webkit-animation-delay: 1.1s;
      animation-delay: 1.1s;
    }

    .rectangle-13 {
      -webkit-animation-delay: 1.2s;
      animation-delay: 1.2s;
    }

    .rectangle-14 {
      -webkit-animation-delay: 1.3s;
      animation-delay: 1.3s;
    }

    .rectangle-15 {
      -webkit-animation-delay: 1.4s;
      animation-delay: 1.4s;
    }

    .rectangle-16 {
      -webkit-animation-delay: 1.5s;
      animation-delay: 1.5s;
    }

    .rectangle-17 {
      -webkit-animation-delay: 1.6s;
      animation-delay: 1.6s;
    }

    .rectangle-18 {
      -webkit-animation-delay: 1.7s;
      animation-delay: 1.7s;
    }

    .rectangle-19 {
      -webkit-animation-delay: 1.8s;
      animation-delay: 1.8s;
    }

    .rectangle-20 {
      -webkit-animation-delay: 1.9s;
      animation-delay: 1.9s;
    }

    .rectangle-21 {
      -webkit-animation-delay: 2s;
      animation-delay: 2s;
    }

    .rectangle-22 {
      -webkit-animation-delay: 2.1s;
      animation-delay: 2.1s;
    }

    .rectangle-23 {
      -webkit-animation-delay: 2.2s;
      animation-delay: 2.2s;
    }

    .rectangle-24 {
      -webkit-animation-delay: 2.3s;
      animation-delay: 2.3s;
    }

    .rectangle-25 {
      -webkit-animation-delay: 2.4s;
      animation-delay: 2.4s;
    }

    .rectangle-26 {
      -webkit-animation-delay: 2.5s;
      animation-delay: 2.5s;
    }

    .rectangle-27 {
      -webkit-animation-delay: 2.6s;
      animation-delay: 2.6s;
    }

    .rectangle-28 {
      -webkit-animation-delay: 2.7s;
      animation-delay: 2.7s;
    }

    .rectangle-29 {
      -webkit-animation-delay: 2.8s;
      animation-delay: 2.8s;
    }

    .rectangle-30 {
      -webkit-animation-delay: 2.9s;
      animation-delay: 2.9s;
    }

    .rectangle-31 {
      -webkit-animation-delay: 3s;
      animation-delay: 3s;
    }

    .rectangle-32 {
      -webkit-animation-delay: 3.1s;
      animation-delay: 3.1s;
    }

    .rectangle-33 {
      -webkit-animation-delay: 3.2s;
      animation-delay: 3.2s;
    }

    .rectangle-34 {
      -webkit-animation-delay: 3.3s;
      animation-delay: 3.3s;
    }

    .rectangle-35 {
      -webkit-animation-delay: 3.4s;
      animation-delay: 3.4s;
    }

    .rectangle-36 {
      -webkit-animation-delay: 3.5s;
      animation-delay: 3.5s;
    }

    .rectangle-37 {
      -webkit-animation-delay: 3.6s;
      animation-delay: 3.6s;
    }

    .rectangle-38 {
      -webkit-animation-delay: 3.7s;
      animation-delay: 3.7s;
    }

    .rectangle-39 {
      -webkit-animation-delay: 3.8s;
      animation-delay: 3.8s;
    }

    div {
      position: relative;
      display: inline-block;
      background: ",
              "#e3fafc",
              ";
      height: 100%;
      width: 10px;
      margin: 0;
      -webkit-animation: load 3s ease-in-out infinite;
      animation: load 3s ease-in-out infinite;
    }
  }

  @-moz-keyframes load {
    0%,
    100% {
      -moz-transform: scaleY(1);
      background: ",
              "#c0eb75",
              ";
    }
    16.67% {
      -moz-transform: scaleY(3);
      background: ",
              "#8ce99a",
              ";
    }
    33.33% {
      -moz-transform: scaleY(1);
      background: ",
              "#63e6be",
              ";
    }
    50% {
      -moz-transform: scaleY(3);
      background: ",
              "#66d9e8",
              ";
    }
    66.67% {
      -moz-transform: scaleY(1);
      background: ",
              "#74c0fc",
              ";
    }
    83.34% {
      -moz-transform: scaleY(3);
      background: ",
              "#91a7ff",
              ";
    }
  }

  @-webkit-keyframes load {
    0%,
    100% {
      -webkit-transform: scaleY(1);
      background: ",
              "#c0eb75",
              ";
    }
    16.67% {
      -webkit-transform: scaleY(3);
      background: ",
              "#8ce99a",
              ";
    }
    33.33% {
      -webkit-transform: scaleY(1);
      background: ",
              "#63e6be",
              ";
    }
    50% {
      -webkit-transform: scaleY(3);
      background: ",
              "#66d9e8",
              ";
    }
    66.67% {
      -webkit-transform: scaleY(1);
      background: ",
              "#74c0fc",
              ";
    }
    83.34% {
      -webkit-transform: scaleY(3);
      background: ",
              "#91a7ff",
              ";
    }
  }

  @keyframes load {
    0%,
    100% {
      transform: scaleY(1);
      background: ",
              "#c0eb75",
              ";
    }
    16.67% {
      transform: scaleY(3);
      background: ",
              "#8ce99a",
              ";
    }
    33.33% {
      transform: scaleY(1);
      background: ",
              "#63e6be",
              ";
    }
    50% {
      transform: scaleY(3);
      background: ",
              "#66d9e8",
              ";
    }
    66.67% {
      transform: scaleY(1);
      background: ",
              "#74c0fc",
              ";
    }
    83.34% {
      transform: scaleY(3);
      background: ",
              "#91a7ff",
              ";
    }
  }
",
            ],
          },
          "displayName": "styled.div",
          "foldedComponentIds": Array [],
          "render": [Function],
          "styledComponentId": "sc-dnqmqq",
          "target": "div",
          "toString": [Function],
          "warnTooManyClasses": [Function],
          "withComponent": [Function],
        },
      },
      "type": Object {
        "$$typeof": Symbol(react.forward_ref),
        "attrs": Array [],
        "componentStyle": ComponentStyle {
          "componentId": "sc-ifAKCX",
          "isStatic": true,
          "rules": Array [
            "
  & {
    width: 100%;
    height: 100%;
    display: block;
    text-align: center;
    margin-top: -15px;
  }
",
          ],
        },
        "displayName": "styled.div",
        "foldedComponentIds": Array [],
        "render": [Function],
        "styledComponentId": "sc-ifAKCX",
        "target": "div",
        "toString": [Function],
        "warnTooManyClasses": [Function],
        "withComponent": [Function],
      },
    },
    "type": Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bxivhb",
        "isStatic": true,
        "rules": Array [
          "
  & {
    width: 100%;
    height: 100%;
    padding: 0.3%;
  }

  &.audio {
    padding-bottom: 0.5%;
  }
  &.button {
    margin-left: -20px;
  }
",
        ],
      },
      "displayName": "styled.div",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bxivhb",
      "target": "div",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    },
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": <ForwardRef>
          <ForwardRef>
            <div
              className="rectangle-5"
            />
            <div
              className="rectangle-4"
            />
            <div
              className="rectangle-3"
            />
            <div
              className="rectangle-2"
            />
            <div
              className="rectangle-1"
            />
            <div
              className="rectangle-2"
            />
            <div
              className="rectangle-3"
            />
            <div
              className="rectangle-4"
            />
            <div
              className="rectangle-5"
            />
          </ForwardRef>
        </ForwardRef>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": <ForwardRef>
            <div
              className="rectangle-5"
            />
            <div
              className="rectangle-4"
            />
            <div
              className="rectangle-3"
            />
            <div
              className="rectangle-2"
            />
            <div
              className="rectangle-1"
            />
            <div
              className="rectangle-2"
            />
            <div
              className="rectangle-3"
            />
            <div
              className="rectangle-4"
            />
            <div
              className="rectangle-5"
            />
          </ForwardRef>,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": Array [
              <div
                className="rectangle-5"
              />,
              <div
                className="rectangle-4"
              />,
              <div
                className="rectangle-3"
              />,
              <div
                className="rectangle-2"
              />,
              <div
                className="rectangle-1"
              />,
              <div
                className="rectangle-2"
              />,
              <div
                className="rectangle-3"
              />,
              <div
                className="rectangle-4"
              />,
              <div
                className="rectangle-5"
              />,
            ],
            "className": undefined,
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "className": "rectangle-5",
              },
              "ref": null,
              "rendered": null,
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "className": "rectangle-4",
              },
              "ref": null,
              "rendered": null,
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "className": "rectangle-3",
              },
              "ref": null,
              "rendered": null,
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "className": "rectangle-2",
              },
              "ref": null,
              "rendered": null,
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "className": "rectangle-1",
              },
              "ref": null,
              "rendered": null,
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "className": "rectangle-2",
              },
              "ref": null,
              "rendered": null,
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "className": "rectangle-3",
              },
              "ref": null,
              "rendered": null,
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "className": "rectangle-4",
              },
              "ref": null,
              "rendered": null,
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "className": "rectangle-5",
              },
              "ref": null,
              "rendered": null,
              "type": "div",
            },
          ],
          "type": Object {
            "$$typeof": Symbol(react.forward_ref),
            "attrs": Array [],
            "componentStyle": ComponentStyle {
              "componentId": "sc-dnqmqq",
              "isStatic": true,
              "rules": Array [
                "
  & {
    height: 0.8em;
    width: 100%;
    display: inherit;
    margin-top: 1.7%;

    &.none {
      display: none;
    }

    .rectangle-2 {
      -webkit-animation-delay: 0.1s;
      animation-delay: 0.1s;
    }

    .rectangle-3 {
      -webkit-animation-delay: 0.2s;
      animation-delay: 0.2s;
    }

    .rectangle-4 {
      -webkit-animation-delay: 0.3s;
      animation-delay: 0.3s;
    }

    .rectangle-5 {
      -webkit-animation-delay: 0.4s;
      animation-delay: 0.4s;
    }

    .rectangle-6 {
      -webkit-animation-delay: 0.5s;
      animation-delay: 0.5s;
    }

    .rectangle-7 {
      -webkit-animation-delay: 0.6s;
      animation-delay: 0.6s;
    }

    .rectangle-8 {
      -webkit-animation-delay: 0.7s;
      animation-delay: 0.7s;
    }

    .rectangle-9 {
      -webkit-animation-delay: 0.8s;
      animation-delay: 0.8s;
    }

    .rectangle-10 {
      -webkit-animation-delay: 0.9s;
      animation-delay: 0.9s;
    }

    .rectangle-11 {
      -webkit-animation-delay: 1s;
      animation-delay: 1s;
    }

    .rectangle-12 {
      -webkit-animation-delay: 1.1s;
      animation-delay: 1.1s;
    }

    .rectangle-13 {
      -webkit-animation-delay: 1.2s;
      animation-delay: 1.2s;
    }

    .rectangle-14 {
      -webkit-animation-delay: 1.3s;
      animation-delay: 1.3s;
    }

    .rectangle-15 {
      -webkit-animation-delay: 1.4s;
      animation-delay: 1.4s;
    }

    .rectangle-16 {
      -webkit-animation-delay: 1.5s;
      animation-delay: 1.5s;
    }

    .rectangle-17 {
      -webkit-animation-delay: 1.6s;
      animation-delay: 1.6s;
    }

    .rectangle-18 {
      -webkit-animation-delay: 1.7s;
      animation-delay: 1.7s;
    }

    .rectangle-19 {
      -webkit-animation-delay: 1.8s;
      animation-delay: 1.8s;
    }

    .rectangle-20 {
      -webkit-animation-delay: 1.9s;
      animation-delay: 1.9s;
    }

    .rectangle-21 {
      -webkit-animation-delay: 2s;
      animation-delay: 2s;
    }

    .rectangle-22 {
      -webkit-animation-delay: 2.1s;
      animation-delay: 2.1s;
    }

    .rectangle-23 {
      -webkit-animation-delay: 2.2s;
      animation-delay: 2.2s;
    }

    .rectangle-24 {
      -webkit-animation-delay: 2.3s;
      animation-delay: 2.3s;
    }

    .rectangle-25 {
      -webkit-animation-delay: 2.4s;
      animation-delay: 2.4s;
    }

    .rectangle-26 {
      -webkit-animation-delay: 2.5s;
      animation-delay: 2.5s;
    }

    .rectangle-27 {
      -webkit-animation-delay: 2.6s;
      animation-delay: 2.6s;
    }

    .rectangle-28 {
      -webkit-animation-delay: 2.7s;
      animation-delay: 2.7s;
    }

    .rectangle-29 {
      -webkit-animation-delay: 2.8s;
      animation-delay: 2.8s;
    }

    .rectangle-30 {
      -webkit-animation-delay: 2.9s;
      animation-delay: 2.9s;
    }

    .rectangle-31 {
      -webkit-animation-delay: 3s;
      animation-delay: 3s;
    }

    .rectangle-32 {
      -webkit-animation-delay: 3.1s;
      animation-delay: 3.1s;
    }

    .rectangle-33 {
      -webkit-animation-delay: 3.2s;
      animation-delay: 3.2s;
    }

    .rectangle-34 {
      -webkit-animation-delay: 3.3s;
      animation-delay: 3.3s;
    }

    .rectangle-35 {
      -webkit-animation-delay: 3.4s;
      animation-delay: 3.4s;
    }

    .rectangle-36 {
      -webkit-animation-delay: 3.5s;
      animation-delay: 3.5s;
    }

    .rectangle-37 {
      -webkit-animation-delay: 3.6s;
      animation-delay: 3.6s;
    }

    .rectangle-38 {
      -webkit-animation-delay: 3.7s;
      animation-delay: 3.7s;
    }

    .rectangle-39 {
      -webkit-animation-delay: 3.8s;
      animation-delay: 3.8s;
    }

    div {
      position: relative;
      display: inline-block;
      background: ",
                "#e3fafc",
                ";
      height: 100%;
      width: 10px;
      margin: 0;
      -webkit-animation: load 3s ease-in-out infinite;
      animation: load 3s ease-in-out infinite;
    }
  }

  @-moz-keyframes load {
    0%,
    100% {
      -moz-transform: scaleY(1);
      background: ",
                "#c0eb75",
                ";
    }
    16.67% {
      -moz-transform: scaleY(3);
      background: ",
                "#8ce99a",
                ";
    }
    33.33% {
      -moz-transform: scaleY(1);
      background: ",
                "#63e6be",
                ";
    }
    50% {
      -moz-transform: scaleY(3);
      background: ",
                "#66d9e8",
                ";
    }
    66.67% {
      -moz-transform: scaleY(1);
      background: ",
                "#74c0fc",
                ";
    }
    83.34% {
      -moz-transform: scaleY(3);
      background: ",
                "#91a7ff",
                ";
    }
  }

  @-webkit-keyframes load {
    0%,
    100% {
      -webkit-transform: scaleY(1);
      background: ",
                "#c0eb75",
                ";
    }
    16.67% {
      -webkit-transform: scaleY(3);
      background: ",
                "#8ce99a",
                ";
    }
    33.33% {
      -webkit-transform: scaleY(1);
      background: ",
                "#63e6be",
                ";
    }
    50% {
      -webkit-transform: scaleY(3);
      background: ",
                "#66d9e8",
                ";
    }
    66.67% {
      -webkit-transform: scaleY(1);
      background: ",
                "#74c0fc",
                ";
    }
    83.34% {
      -webkit-transform: scaleY(3);
      background: ",
                "#91a7ff",
                ";
    }
  }

  @keyframes load {
    0%,
    100% {
      transform: scaleY(1);
      background: ",
                "#c0eb75",
                ";
    }
    16.67% {
      transform: scaleY(3);
      background: ",
                "#8ce99a",
                ";
    }
    33.33% {
      transform: scaleY(1);
      background: ",
                "#63e6be",
                ";
    }
    50% {
      transform: scaleY(3);
      background: ",
                "#66d9e8",
                ";
    }
    66.67% {
      transform: scaleY(1);
      background: ",
                "#74c0fc",
                ";
    }
    83.34% {
      transform: scaleY(3);
      background: ",
                "#91a7ff",
                ";
    }
  }
",
              ],
            },
            "displayName": "styled.div",
            "foldedComponentIds": Array [],
            "render": [Function],
            "styledComponentId": "sc-dnqmqq",
            "target": "div",
            "toString": [Function],
            "warnTooManyClasses": [Function],
            "withComponent": [Function],
          },
        },
        "type": Object {
          "$$typeof": Symbol(react.forward_ref),
          "attrs": Array [],
          "componentStyle": ComponentStyle {
            "componentId": "sc-ifAKCX",
            "isStatic": true,
            "rules": Array [
              "
  & {
    width: 100%;
    height: 100%;
    display: block;
    text-align: center;
    margin-top: -15px;
  }
",
            ],
          },
          "displayName": "styled.div",
          "foldedComponentIds": Array [],
          "render": [Function],
          "styledComponentId": "sc-ifAKCX",
          "target": "div",
          "toString": [Function],
          "warnTooManyClasses": [Function],
          "withComponent": [Function],
        },
      },
      "type": Object {
        "$$typeof": Symbol(react.forward_ref),
        "attrs": Array [],
        "componentStyle": ComponentStyle {
          "componentId": "sc-bxivhb",
          "isStatic": true,
          "rules": Array [
            "
  & {
    width: 100%;
    height: 100%;
    padding: 0.3%;
  }

  &.audio {
    padding-bottom: 0.5%;
  }
  &.button {
    margin-left: -20px;
  }
",
          ],
        },
        "displayName": "styled.div",
        "foldedComponentIds": Array [],
        "render": [Function],
        "styledComponentId": "sc-bxivhb",
        "target": "div",
        "toString": [Function],
        "warnTooManyClasses": [Function],
        "withComponent": [Function],
      },
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
